src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -1205,12 +1204,13 @@ subroutine KPP_compute_BLD(CS, G, GV, US, h, Temp, Salt, u, v, tv, uStar, buoyFl
             endif
 
           enddo
-          surfTemp = surfHtemp / hTot
-          surfSalt = surfHsalt / hTot
-          surfU    = surfHu    / hTot
-          surfV    = surfHv    / hTot
-          surfUs   = surfHus   / hTot
-          surfVs   = surfHvs   / hTot
+          I_hTot = 1./hTot
```

Comment:
@gustavo-marques While this is better for performance, it changes answers, soI reverted it in the previous PR. Are we okay with answer changes, or should we revert this again?

Reply:
Yes, we are okay with answer changes. Thanks!